locale/foreman_remote_execution.pot

```diff
@@ -2075,73 +2073,73 @@ msgid "Open in new tab"
 msgstr ""
 
 #:
-#: ../webpack/JobInvocationDetail/TemplateInvocationComponents/PreviewTemplate.js:41
+#: ../webpack/JobInvocationDetail/TemplateInvocationComponents/PreviewTemplate.js:34
 msgid "No user input"
 msgstr ""
 
 #:
-#: ../webpack/JobInvocationDetail/TemplateInvocationComponents/TemplateActionButtons.js:35
+#: ../webpack/JobInvocationDetail/TemplateInvocationComponents/TemplateActionButtons.js:36
 msgid "Cancel Task"
 msgstr ""
 
 #:
-#: ../webpack/JobInvocationDetail/TemplateInvocationComponents/TemplateActionButtons.js:42
+#: ../webpack/JobInvocationDetail/TemplateInvocationComponents/TemplateActionButtons.js:43
 msgid "Trying to cancel the task for the host"
 msgstr ""
 
 #:
-#: ../webpack/JobInvocationDetail/TemplateInvocationComponents/TemplateActionButtons.js:50
+#: ../webpack/JobInvocationDetail/TemplateInvocationComponents/TemplateActionButtons.js:51
 msgid "Task for the host cancelled succesfully"
 msgstr ""
 
 #:
-#: ../webpack/JobInvocationDetail/TemplateInvocationComponents/TemplateActionButtons.js:58
-msgid "Abort task"
+#: ../webpack/JobInvocationDetail/TemplateInvocationComponents/TemplateActionButtons.js:59
+msgid "Abort Task"
```

Comment:
I wouldn't expect this line because of d057523a7e0fa3b751b5b82c9275ce11a64412ab

Reply:
Sigh, pebcak